### PR TITLE
Mybranch

### PR DESCRIPTION
**Pull request by Saugat-ghimire**

### CODE REVIEW

File: src/main/java/org/example/algorithms/BubbleSort.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// import the Class
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Main {
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/algorithms/BubbleSort.java:5]: Change class name to BubbleSort

File: src/main/java/org/example/algorithms/BubbleSort.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  // create an object of scanner
+  // to take input from the user
+  Scanner input = new Scanner(System.in);
+
+  // method to perform bubble sort
+  void BubbleSort(int array[]) {
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/algorithms/BubbleSort.java:12]: use camelCase i.e. _bubbleSort_

File: src/main/java/org/example/algorithms/BubbleSort.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    int size = array.length;
+
+    // for ascending or descending sort
+    System.out.println("Choose Sorting Order:");
+    System.out.println("1 for Ascending \n2 for Descending");
+    int SortOrder = input.nextInt();
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/algorithms/BubbleSort.java:18]: camelCase - sortOrder

**Review by BimalRajGyawali — CHANGES_REQUESTED** [review]
Follow naming convention https://github.com/BimalRajGyawali/hacktoberfest-java/blob/main/CONTRIBUTING.md#naming-conventions

File: src/main/java/org/example/algorithms/BubbleSort.java (hunk 2 above)
Comment (Saugat-ghimire) [reply src/main/java/org/example/algorithms/BubbleSort.java:12]: sure !

**Pull request closed without merging**